buildenv/jenkins/docker-slaves/x86/centos6.9/Dockerfile

```diff
@@ -315,6 +292,38 @@ RUN yum -y update \
   && cd .. \
   && rm -rf Python-3.7.3
 
+# Any commands that require gcc need to be run before this SHELL command and after the previous SHELL
+SHELL ["/bin/sh","-c"]
```

Comment:
The following steps didn't *need* to be relocated below.

Reply:
During the section where we change the `SHELL` command to use a login shell, the `USER` variable we set gets overwritten to `root`. By moving these steps outside of the login shell section, they will see the `USER` variable as being set to `jenkins`. I will add a comment to indicate this.